modules/components/tag/tag.component.ts

```diff
@@ -332,6 +344,7 @@ export class TagComponent {
      * @param input
      */
     private storeNewValue(input: string): void {
+        console.log('aaaaa Tag.storeNewValue', input);
```

Comment:
@huyld Plz remove `console.log`

Reply:
@tronghiant I've removed it. Sorry for my carelessness 🙇

Reply:
@tronghiant I've removed it. Sorry for my carelessness 🙇